tests/by-util/test_factor.rs

```diff
@@ -1258,3 +1258,8 @@ const PRIMES50: &[u64] = &[
     1125899906841623,
     1125899906841613,
 ];
+
+#[test]
+fn fails_on_directory() {
+    new_ucmd!().arg("<").arg(".").fails();
```

Comment:
This fails as it should, but not for the reason you're expecting. The `<` will not pipe here. It will be an argument to `factor`, so `factor '<' '.'`.

Reply:
ah ah, fun :)